Cabal/Distribution/Simple/Haddock.hs

```diff
@@ -224,6 +224,7 @@ haddock pkg_descr lbi suffixes flags = do
                          version
               let libArgs' = commonArgs `mappend` libArgs
               runHaddock verbosity tmpFileOpts comp confHaddock libArgs'
+        CPLib  _ -> putStrLn "<<haddock CPLib>>"
```

Comment:
Oh, whoops. Sorry about that. Will fix that (along with your other comments). Not yet sure why Travis is failing though.